### DISCUSSION TITLE
the addition of a new loss term

### DISCUSSION THREAD

**Discussion by Fankunkun123** [lululxvi/deepxde#1616] [Q&A]
Hello, Professor  @lululxvi  and other professors.
At present, I have encountered some problems in using deepXDE library, please help to solve them. Thank you very much. 
> Question 1: When using deepXDE, what should I do if I want to add a new loss term to the total loss (the added loss term is different from the gradient boosting term in the gPINN article, the loss term I want to add is the loss that does not have the first and second derivatives of the equation, but is constructed using other arithmetic methods). ?

For example, if we want to add a new Loss to the total loss: L1=ax+bx+cx; Total Loss=Lpde+Lbc+L1.

I hope you can help me solve the problem, thank you very much .

**Comment by lululxvi** [comment]
The same. Just add the loss.
